experiment/test_config.yaml

```diff
@@ -763,6 +763,78 @@ jobs:
     args:
     - --gcp-project=ubuntu-image-validation
     sigOwners: ['sig-node']
+  # release branch jobs (stable2 and stable3)
+  ci-kubernetes-e2e-gce-gci-k8sstable3-alphafeatures:
+    interval: 6h
+    args:
+    - --env=KUBE_FEATURE_GATES=AllAlpha=true
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-alphafeatures:
+    interval: 2h
+    args:
+    - --env=KUBE_FEATURE_GATES=AllAlpha=true
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-ingress:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-ingress:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-reboot:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-reboot:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-default:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-default:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-serial:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-serial:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable3-slow:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gce-gci-k8sstable2-slow:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable3-ingress:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable2-ingress:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable3-reboot:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable2-reboot:
+    interval: 2h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable3-default:
+    interval: 6h
+    sigOwners: ['sig-gcp']
+  ci-kubernetes-e2e-gke-gci-k8sstable2-default:
```

Comment:
The above comment does not apply to GKE tests since the specific image used in tests are specified in GKE code, so I guess we can just use use `cos` here.